cmd/pg/catchup_fetch.go

```diff
@@ -8,7 +8,9 @@ import (
 
 const (
 	CatchupFetchShortDescription = "Fetches an incremental backup from storage"
+	UseNewUnwrapDescription      = "New implementation of catchup unwrap (testing)"
```

Comment:
beta feature

Reply:
Fixed in 1659a812fdd448d376f9947fcc579956e006e151